dozer-lambda/src/js/tests.rs

```diff
@@ -0,0 +1,206 @@
+use std::{pin::pin, time::Duration};
+
+use deno_runtime::deno_core::futures::future::{join, select, Either};
+
+use super::*;
+
+#[test]
+fn test_lambda_runtime() {
+    // env_logger::init();
+    let tokio_runtime = Arc::new(
+        tokio::runtime::Builder::new_current_thread()
+            .enable_all()
+            .build()
+            .unwrap(),
+    );
+    tokio_runtime.block_on(test_lambda_runtime_impl(tokio_runtime.clone()));
+}
+
+async fn test_lambda_runtime_impl(tokio_runtime: Arc<tokio::runtime::Runtime>) {
+    let (app_url, app_server) = mock::start_mock_internal_pipeline_server().await;
+    let lambda_modules = vec![JavaScriptLambda {
+        endpoint: mock::mock_endpoint(),
+        module: "src/js/test_lambda.js".to_string(),
+    }];
+    let lambda_runtime = Runtime::new(tokio_runtime, app_url, lambda_modules, Default::default());
```

Comment:
Can we add a layer of abstraction to `Runtime`/`Trigger` so that we can test it without setting up a whole `InternalPipelineServer`? I'd like it if we could just do `Trigger::add_lambda(op_receiver: spsc::Receiver, schema: Schema)`.

Reply:
If we add that, there's basically no code left in `Trigger` to be tested.